src/main/java/pages/FacturaVanzari.java

```diff
@@ -0,0 +1,176 @@
+
+    @FindBy(xpath = "//input[@id='input-articleFormData.quantity']")
+    private WebElement cantitate;
+    @FindBy(xpath = "//input[@id='input-articleFormData.selling_price']")
+    private WebElement pretUnitar;
+    @FindBy(xpath = "//input[@id='article-form-data-total-selling-price']")
+    private WebElement total;
+   @FindBy(xpath = "//span[contains(text(),'Adaugă articol')]")
+    private WebElement adaugaArticol;
+   @FindBy(xpath = "//button[contains(text(),'Salvează și vezi')]")
+   private WebElement salveazaSiVezi;
+
+
+
+
+    public String[] expectedAlertMessages = {"Datele de autentificare introduse nu se potrivesc cu baza de date.",
+            "Campul \"E-mail\" este obligatoriu.",
+            "Campul \"Parola\" este obligatoriu."
+    };
+
+
+    public void addText(WebElement emailField, String email) {
+    }
+
+    public void addPassword(String password) {
+        addText(passwordField, password);
+    }
+
+    public void clickRememberMeCheckbox() {
+        click(rememberMeCheckbox);
+    }
+
+    public void click(WebElement rememberMeCheckbox) {
+    }
+
+    public void clickLoginButton() {
+        click(loginButton);
+    }
+
+    public void clickAuthenticationButton() {
+        click(authenticationButton);
+        Assert.assertEquals(driver_local.getCurrentUrl(), "https://account.contapp.ro/login");
+    }
+
+    public void clickCreateAccountButton() {
+        click(createAccountButton);
+        Assert.assertEquals(driver_local.getCurrentUrl(), "https://account.contapp.ro/register",
+                "The url page  is not the expected one.");}
+ public void clickcontAppPs(){click(contAppPs);}
+public void clickconfigurare(){click(configurare);}
+    public void clickvenituri() {
+        click(venituri);
+    }
+
+    public void clickfacturiVanzari() {click(facturiVanzari);
+    }
+
+
+    public void clickClienti() {
+        click(clienti);
+    }
+    public void adaugaFacturiVanzari() {click(facturiVanzari);}
+
+    public void alegeClient() {
+        click(alegeClient);
+    }
+
+    public void alegeTip() {
+        click(alegeTip);
+    }
+
+    public void serieFactura() {
+        click(serieFactura);
+    }
+
+    public void numarFactura() {
+        click(numarFactura);
+    }
+
+    public void tipArticol() {
+        click(tipArticol);
+    }
+
+    public void denumireArticol() {
+        click(denumireArticol);
+    }
+
+
+    public void cantitate() {
+        click(cantitate);
+    }
+
+    public void pretUnitar() {
+        click(pretUnitar);
+    }
+
+    public void total() {
+        click(total);
+    }
+
+    public void adaugaArticol() {click(adaugaArticol);}
+    public void salveazaSiVezi(){click(salveazaSiVezi);}
+
+    public void verifyAlertMessages() {
+        if (emailAlertMessage.getText().equals(expectedAlertMessages[0]) && passwordAlertMessage.getText().equals(expectedAlertMessages[0])) {
+            System.out.println(emailAlertMessage.getText());
```

Comment:
Alertele se verifica cu Assert, **fara System.out.println** in code , il folositi numai la debugging

Reply:
OK. Multumesc

Reply:
am sters comentariile din cod.